### DISCUSSION TITLE
Question regarding "torch.manual_seed"

### DISCUSSION THREAD

**Discussion by Reetos** [mrdbourke/pytorch-deep-learning#358] [Q&A] [answered]
In line 5 (torch.random.manual_seed(seed=RANDOM_SEED) # try commenting this line out and seeing what happens
), why do you need "random" in "torch.random.manual_seed"?
The result of the comparison between random_tensor_C and random_tensor_D does not change without "random".

```
import torch
import random

RANDOM_SEED=42 # try changing this to different values and see what happens to the numbers below
torch.manual_seed(seed=RANDOM_SEED) 
random_tensor_C = torch.rand(3, 4)

torch.random.manual_seed(seed=RANDOM_SEED) # try commenting this line out and seeing what happens
random_tensor_D = torch.rand(3, 4)

print(f"Tensor C:\n{random_tensor_C}\n")
print(f"Tensor D:\n{random_tensor_D}\n")
print(f"Does Tensor C equal Tensor D? (anywhere)")
random_tensor_C == random_tensor_D
```

**Comment by mrdbourke** [comment, marked as answer]
Hi @Reetos,

You're right, you don't need `torch.random.manual_seed()`. 

You can achieve the full functionality with `torch.manual_seed()`, see the documentation here: https://pytorch.org/docs/stable/generated/torch.manual_seed.html

If you feel the code should be updated to reflect this, please feel free to leave a pull request :)

**Reply by Reetos** [reply]
Thank you!
